CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Castle Core Changelog
 
+## Next
+
+Enhancements:
+- Now target frameworks are: .NET 8, .NET Framework 4.6.2, .NET Standard 2.0
+- Tests platforms now are: .NET Framework 4.6.2, .NET 8, .NET 9
+- Lang version now is 13
```

Comment:
Not sure if this needs to be in the changelog, as it typically won't affect consumers of the `Castle.Core` package.

It's a possibly important detail to contributors though, so we should mention the updated C# language version somewhere. Probably in the `README`'s "Building" section, which currently still says C# 9.